src/App.js

```diff
@@ -1,11 +1,24 @@
 import React, { Component } from 'react'
 import './App.css'
+import Light from './components/Light'
+import Header from './components/Header'
+import Footer from './components/Footer'
 
 class App extends Component{
   render(){
     return(
       <>
-        <h1>Hello World!</h1>
+        <h1>Hello Echo!</h1>
+        <br></br>
+        <br></br>
+        <Header />
+        <Light />
+      <br></br>
+      <br></br>
```

Comment:
I wonder if there is some css that you could add to your footer and other components so you don't have to use so many <br>